conf/broker.yaml.template

```diff
@@ -4,3 +4,17 @@ BROKER:
     HOST_WORKFLOWS:
         POWER_CONTROL: vm-power-operation
         EXTEND: extend-vm
+    # Client release versions and confs
+    CLIENTS:
```

Comment:
I think this config should go in `content_host.yaml`, as that's where we should be defining versions, workflows, and other settings data concerning content hosts.

Reply:
Moved to `content_host.yaml`.